{"name": "test_add[ni-3-error]", "status": "passed", "attachments": [{"name": "stdout", "source": "9ba7066e-363d-42f9-9264-b22abce452f4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "'ni'"}, {"name": "b", "value": "3"}, {"name": "expecter", "value": "'error'"}], "start": 1615047822425, "stop": 1615047822426, "uuid": "bb55ff4f-9ff9-40e1-9c09-32fe7e30d116", "historyId": "8217f036d2d610d7189dec24013d2e2a", "testCaseId": "46313a82a5980d1f654f624e59d2d766", "fullName": "test_work_two.TestCalc#test_add", "labels": [{"name": "tag", "value": "run(order=0)"}, {"name": "tag", "value": "first"}, {"name": "suite", "value": "test_work_two"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "192.168.0.102"}, {"name": "thread", "value": "4427-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work_two"}]}